### ISSUE TITLE
The background of the website can be changed a bit to improve the readability.

### ISSUE THREAD

**Issue by aanchalrakheja** [ALPHAVIO/WordNook#959]
### **Describe the bug**
Some text merges with the background of the website hence making it difficult to read. The background of the website can be changed to improve the readability.
<hr/>

**Note:**
* Do mention if your are working under GSSoC'21.
* If you want to work on an issue, you should check if it has already been assigned to anyone. **If the issue is free** you can comment `/assign` to get the issue assigned to you.
* If you are raising a new issue and want to work on it then also you should comment `/assign` under the issue to get it auto assigned.
* Please **refrain from** adding labels to your issue/pull-request on your own. It is the job of the Project Admin and the Mentors to review your issue/pull-request and add labels accordingly.
*  According to the **level** of the issue, respective ***time*** will be allotted under which the task needs to be completed/resolved by the assigned person. If the issue is not resolved within the given time, it will be assigned to the next person. In case of any discrepancies, the decision of the maintainer will be final.
```
level - 0: 2 days
level - 1: 3 days
level - 2: 4 days
level - 3: 5 days
```

**Comment by aanchalrakheja** [comment]
i would like to work on this issue

**Comment by aanchalrakheja** [comment]
![background](https://user-images.githubusercontent.com/68388581/120082777-6be14780-c0e2-11eb-8115-5a5434440782.png)
This background would give it a look of a crumbled paper.

**Comment by keshavgbpecdelhi** [comment]
seems good

**Comment by aanchalrakheja** [comment]
![image](https://user-images.githubusercontent.com/68388581/120102616-7abf0d00-c169-11eb-9e21-18e646925a17.png)
![image](https://user-images.githubusercontent.com/68388581/120102628-8ca0b000-c169-11eb-9ef1-f0aa0a45423d.png)
This is how it looks.

**Comment by keshavgbpecdelhi** [comment]
It's looking nice, but don't u think it's like the dark theme, which is already there🤔.
Let's see what @Pranav016 thinks.

**Comment by Pranav016** [comment]
Changes look fine but do we really need it ? Go through #717 
Let me know if this is still required.

**Comment by aanchalrakheja** [comment]
Even after #717 being merged, there are places where readability of the text can be better. Its not like text is not at all visible but at one glance it seems to merge with the background which I think should be prevented for the text to stand out. 
Rest how so ever u suggest about it's need.

**Comment by Pranav016** [comment]
> Even after #717 being merged, there are places where readability of the text can be better. Its not like text is not at all visible but at one glance it seems to merge with the background which I think should be prevented for the text to stand out.
> Rest how so ever u suggest about it's need.

I checked the screenshot you added above with the new background, the text definitely looks better. I think you can go ahead with the changes.

**Comment by aanchalrakheja** [comment]
ok sure

**Comment by aanchalrakheja** [comment]
![image](https://user-images.githubusercontent.com/68388581/120118376-d90ede80-c1af-11eb-8666-1da16c81531e.png)
Can u help me solve this error. 
Line 21 of darkmode.js says 
![image](https://user-images.githubusercontent.com/68388581/120118397-f643ad00-c1af-11eb-906a-f65313052c50.png)
I am not able to locate the element with the id="blog-image"

**Comment by aanchalrakheja** [comment]
![image](https://user-images.githubusercontent.com/68388581/120118430-1ecba700-c1b0-11eb-99fc-178a9080a9b9.png)
Can u help me solve this error.
The line 21 of darkmode.js says:
![image](https://user-images.githubusercontent.com/68388581/120118448-34d96780-c1b0-11eb-9529-1ecd8ee65032.png)
I am not able to locate the element with id "blog-image"

**Comment by Pranav016** [comment]
> I am not able to locate the element with id "blog-image"

Use the search all feature in your VS code, searches for a particular thing among all the files. I don't remember correctly but I think its shortcut was `Ctrl + Shift + F` but you can also google it.
